src/query/ee_features/resources_management/src/resources_management.rs

```diff
@@ -96,31 +96,31 @@ impl ResourcesManagement for DummyResourcesManagement {
     }
 
     async fn create_warehouse(&self, _: String, _: Vec<SelectedNode>) -> Result<WarehouseInfo> {
-        Err(ErrorCode::Unimplemented("The use of this feature requires a Databend Enterprise Edition license. To unlock enterprise features, please contact Databend to obtain a license. Learn more at https://docs.databend.com/guides/overview/editions/dee/"))
```

Comment:
how about creating a global str variable

Reply:
Yes, that makes more sense.